tests/phpunit/tests/formatting/date.php

```diff
@@ -222,13 +233,18 @@ public function test_is8601_to_datetime_correct_time( $timezone_string, $gmt_off
 	public function timezone_provider() {
 		return array(
 			array(
-				'timezone_string' => 'Europe/Kiev',
+				'timezone_string' => 'Europe/Helsinki',
 				'gmt_offset'      => 3,
 			),
 			array(
 				'timezone_string' => '',
 				'gmt_offset'      => 3,
 			),
+			// @ticket 56468.
+			'Deprecated timezone string and no gmt offset set' => array(
```

Comment:
```suggestion
			'Deprecated timezone string and no GMT offset set' => array(
```
Also, genuine question about something trivial:

Should datasets begin with uppercase, or lowercase?

```
is8601 to datetime correct time with Deprecated timezone string and no GMT offset set
versus
is8601 to datetime correct time with deprecated timezone string and no GMT offset set
```

Reply:
There is currently no standard set for this. I normally would use sentence case, though it depends on the codebase.